### PR TITLE
IPFS pinning instructions

### PR DESCRIPTION
**Pull request by mattetre**
Basic IPFS pinning instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +31,45 @@ npm run dev
...
+
+#### What is Pinning?
+
+Pinning ensures the generator interface remains accessible through the IPFS network. The more people who pin the content, the more resilient and decentralized the network becomes.
+
+#### Pin Using Pinata
+
+1. Create an account at pinata.cloud
+2. Navigate to the "Pin By CID" page
+3. Enter CID: `QmYr3B1TREbgbS29hWmEAcnU96msFoGJ18mYemLtPkj7T6`
```
Comment (ryley-o) [inline README.md]: edit - 
I think we should describe option to pin by CID, or using a service like pinata, should be able to auto-catch updates via "Pin by DNSLink."

**Review by ryley-o — APPROVED** [review]
lgtm, pending confirmation of CID and confirming ipns strategy

File: README.md [hunk 2]
```diff
@@ -28,3 +31,45 @@ npm run dev
...
+The application is hosted on centralized servers as well as on the decentralized IPFS network.
+
+### Pin the On-Chain Generator
+
+Help support decentralized access to the Art Blocks on-chain generator by pinning it to IPFS.
+
+IPFS detail:
+
+- CID: `QmYr3B1TREbgbS29hWmEAcnU96msFoGJ18mYemLtPkj7T6`
+- IPNS: `onchain-generator.artblocks.io`
```
Comment (yoshiwarab) [inline README.md]: I think this should say DNSLink instead of IPNS

**Review by yoshiwarab — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -28,3 +31,53 @@ npm run dev
...
+
+#### What is Pinning?
+
+Pinning ensures the generator interface remains accessible through the IPFS network. The more people who pin the content, the more resilient and decentralized the network becomes.
+
+#### Pin Using Pinata
+
+1. Create an account at pinata.cloud
+2. Navigate to the "Pin By CID" page
+3. Enter CID: `QmYr3B1TREbgbS29hWmEAcnU96msFoGJ18mYemLtPkj7T6`
```
Comment (yoshiwarab) [inline README.md]:
```suggestion
3. Enter CID: `bafybeicodqh762wabjalew7ysiprzsb5cpbqdxtkbtffpgj25ep2bkfluu `
```

**Pull request merged**